scripts/start_geth_nodes.sh

```
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

For these test scripts which don't have to hide failure, my preference is that they shouldn't, along the lines of `set -eu` (or one of its variations (`set -euo pipefail`, etc). Sometimes this disallows genuinely useful tricks, and loses too much expressive power, but it's also good for catching, in this kind of setup, if there's some change which misaligns variable names or similar.

---

Added